src/library/autodj/autodjprocessor.cpp

```diff
@@ -1659,3 +1678,83 @@ bool AutoDJProcessor::nextTrackLoaded() {
 
     return loadedTrack == getNextTrackFromQueue();
 }
+
+double AutoDJProcessor::getFadeTime() {
+    // TODO(c3n7): Is there a better place for this? Find out.
+    // Copied from syncable.h
```

Comment:
If it is already on a file that you can include, why do you copy its content?

Reply:
[Done](https://github.com/mixxxdj/mixxx/pull/2746/commits/c2f9ad507c0407d0d8ab45b0403873cafd6fba8a). Included `syncable.h`.